src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -127,6 +128,7 @@ def app_create(cmd, client, resource_group, service, name,
         instance_count=instance_count,
         environment_variables=env,
         jvm_options=jvm_options,
+        net_core_main_entry_path=net_core_main_entry_path,
         runtime_version=runtime_version,)
     user_source_info = models.UserSourceInfo(
         relative_path='<default>', type='Jar')
```

Comment:
It's completely OK to pass `<default>` `JAR` to .net core app (deployment), but might be weird for users using REST API calls.